src/Orchard.Web/Modules/Orchard.MediaLibrary/Handlers/MediaLibraryPickerFieldHandler.cs

```diff
@@ -37,7 +37,8 @@ public class MediaLibraryPickerFieldHandler : ContentHandler {
 
             foreach (var field in fields) {
                 var localField = field;
-                localField._contentItems = new Lazy<IEnumerable<MediaPart>>(() => _contentManager.GetMany<MediaPart>(localField.Ids, VersionOptions.Published, QueryHints.Empty).ToList());
```

Comment:
Should the local _contentManager be removed then?

Reply:
As you suggested, _contentManager variable has been removed from both ContentPickerFieldHandler and MediaLibraryPickerFieldHandler.